CMakeLists.txt

```diff
@@ -395,12 +398,18 @@ install(
         ${DATA_INSTALL_DIR}/${PROJECT_NAME}/cmake
     )
 
-# Install dependencies
+# Install dependencies.
 if(EXISTS ${CMAKE_BINARY_DIR}/temp_install/)
-    install(
-        DIRECTORY
-            ${CMAKE_BINARY_DIR}/temp_install/
-        DESTINATION
-            ${CMAKE_INSTALL_PREFIX}
+   foreach(dependency ${_deps})
+        list(GET dependency 0 _name)
+        list(GET dependency 1 _version)
+        install(
+            DIRECTORY
+                ${CMAKE_BINARY_DIR}/temp_install/${_name}-${_version}
+            DESTINATION .
```

Comment:
I prefer the `${CMAKE_INSTALL_PREFIX}`, what do you think?

Reply:
I am not sure if using `${CMAKE_INSTALL_PREFIX}`  invalidates de use of `DESTDIR` we should check.